### ISSUE TITLE
for WSA build typo?

### ISSUE THREAD

**Issue by tomlee-dev** [CesiumGS/cesium-unity#366]
I found that akward code for building WSA (for editor.)
https://github.com/CesiumGS/cesium-unity/blob/878245e15f2165d7bf331788fcfa69380fe1bc89/Editor/ConfigureReinterop.cs#L26

I think this line have to be...

- public const string CppOutputPath = "../native~/Editor/generated-WSA";

than 

- public const string CppOutputPath = "../native~/Runtime/generated-WSA";

**Comment by kring** [comment]
You're right! It didn't break anything because the Editor assembly is never built for WSA / UWP.

**Comment by j9liu** [comment]
I believe this is outdated as of #642.
